src/handlers/genreById.ts

```diff
@@ -0,0 +1,41 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import getAll from '../models/Genre/getAll';
+import { GenreResponse } from '../types/apiResponse';
+
+export const handler: APIGatewayProxyHandler = async (event) => {
+	const notFoundResponse = {
+		statusCode: 404,
+		body: JSON.stringify({ message: 'Not Found' }),
+	};
+	const genreId = event?.pathParameters?.id;
+
+	if (!genreId) {
+		return notFoundResponse;
+	}
+
+	try {
+		const response = await getAll({ query: { id: genreId } });
```

Comment:
Should this actually be a get by ID function so yu can remove the `response.data[0]` from below and hide that behind the specific implementation?

Reply:
Good call, created a `getById` function